mypy.ini

```diff
@@ -5,6 +5,11 @@ exclude = (?x)(
           docs|
           \.run|
           \.tox|
+          \.nox|
+          env|
+          .env|
+          venv|
+          \.venv|
```

Comment:
...since mypy found errors in some libraries contained in a conda environment from a few years ago on my old work laptop.